conans/model/build_info.py

```diff
@@ -44,18 +44,21 @@ def __init__(self, root_folder):
         self.libdirs.append(DEFAULT_LIB)
         self.bindirs.append(DEFAULT_BIN)
         self.resdirs.append(DEFAULT_RES)
+        self.abs_includedirs = []
+        self.abs_libdirs = []
+        self.abs_bindirs = []
```

Comment:
Why not append to the same libdirs, bindirs and resdirs? you can detect if its an absolute path with os.path.isabs